packages/expansion-hub/src/MotorMode.ts

```diff
@@ -0,0 +1,7 @@
+//The order of these should match the RHSPlib documentation. We rely on the
+//numerical value matching.
+export enum MotorMode {
+    CONSTANT_POWER,
+    CONSTANT_VELOCITY,
+    TARGET_POSITION,
```

Comment:
The SDK uses horrible names for constant power and constant velocity currently, and there was discussion of what we should rename them to here: https://github.com/FIRST-Tech-Challenge/ftc_sdk/issues/1990

I don't love `TARGET_POSITION`. I think I do like Chris's suggestion of `UNREGULATED`, `REGULATED_VELOCITY`, `REGULATED_POSITION`. "Constant" implies that it does a perfect job of maintaining the desired value.

Reply:
How does OPEN_LOOP sound as a replacement for CONSTANT_POWER? This isn't the FTC SDK, so we don't have the same assumptions about user knowledge.

Reply:
I'm fine with that.